### PR TITLE
Timeout bugs

### PR DESCRIPTION
**Pull request by blakebyrnes**
- Mitm could throw unhandled errors during write if the stream got closed
- Tab DomContentLoaded can take a really long time if you loop and re-load in the same tab a lot of times. Extended the timeout to 60 seconds

### CODE REVIEW

File: commons/SourceMapSupport.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
-import * as SourceMapSupport from 'source-map-support';
+import { install } from 'source-map-support';
 
 if (!Error[Symbol.for('source-map-support')]) {
   Error[Symbol.for('source-map-support')] = true;
-  SourceMapSupport.install({ uncaughtShimInstalled: false });
+  install({ handleUncaughtExceptions: false });
```
Comment (calebjclark) [inline commons/SourceMapSupport.ts:5]: I find that destructured method/property imports to be much harder to follow in the code base. I realize that this allows for smaller files, but does it matter for a server-side tool like SA?

File: core/lib/TabNavigationsObserver.ts [hunk 2]
```diff
@@ -221,7 +221,7 @@ export default class TabNavigationsObserver {
     if (this.waitingForPromise) this.cancelWaiting('New location trigger created');
 
     this.waitingForStatus = status;
-    this.waitingForPromise = createPromise<void>(timeoutMs ?? 30e3);
+    this.waitingForPromise = createPromise<void>(timeoutMs ?? 60e3);
```
Comment (calebjclark) [inline core/lib/TabNavigationsObserver.ts:224]: double the default timeout?

File: commons/SourceMapSupport.ts (hunk 1 above)
Comment (blakebyrnes) [reply commons/SourceMapSupport.ts:5]: No, it doesn't really matter, but in this specific case, source map support was behaving strangely with the es export * and nothing else is happening in this file, so I thought an exception was ok.

File: core/lib/TabNavigationsObserver.ts (hunk 2 above)
Comment (blakebyrnes) [reply core/lib/TabNavigationsObserver.ts:224]: DOMContentLoaded has been recorded taking 34 and 32 seconds when I was hitting the same url over and over.... so I moved to 60. I don't like having a timeout here with a specific value, but it kind of needs to be something.

File: commons/SourceMapSupport.ts (hunk 1 above)
Comment (calebjclark) [reply commons/SourceMapSupport.ts:5]: Cool. That's a good reason.

**Pull request merged**